src/search/landmarks/util.cc

```diff
@@ -12,7 +13,8 @@ namespace landmarks {
 static bool condition_is_reachable(
     const ConditionsProxy &conditions, const vector<vector<bool>> &reached) {
     for (FactProxy condition : conditions) {
-        if (!reached[condition.get_variable().get_id()][condition.get_value()]) {
+        if (!reached[condition.get_variable().get_id()]
+                    [condition.get_value()]) {
```

Comment:
Not sure how I feel about this. I struggled to accept this at first but after a bit of time I think it's fine and actually I wouldn't know a better name (except maybe introducing variable names for intermediate parts).